### PR TITLE
[IMG-139] feat: android builds

### PR DESCRIPTION
**Pull request by ryanbourdais**
Adds an android playbook and tasks for building the android machine image. This is a part of the push for automating image creation.

### CODE REVIEW

File: manifest/android-test.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (JalexChen) [inline manifest/android-test.json:1]: are we keeping this file here or in the other repo?

File: roles/android/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+- name: install circle android and ensure circleci owns all packages
+  become: true
+  become_method: sudo
+  ansible.builtin.shell: |
+    cp /tmp/circleci-provisioner/circle-android /usr/local/bin/
+    sudo chown -R {{ circleci_user }}:{{ circleci_user }} /usr/local/bin/circle-android
+    sudo chmod +x /usr/local/bin/circle-android
+    chown -R {{ circleci_user }}:{{ circleci_user }} /opt/android/sdk
+
+# - name: set JAVA HOME and JDK HOME
```
Comment (JalexChen) [inline roles/android/tasks/main.yml:86]: i would remove this if we aren't using tit

File: manifest/android-test.json (hunk 1 above)
Comment (JalexChen) [reply manifest/android-test.json:1]: just a note, some of the versions are outdated if we end up releasing a build

**Review by JalexChen — COMMENTED** [review]
left some comments

File: manifest/android-test.json (hunk 1 above)
Comment (ryanbourdais) [reply manifest/android-test.json:1]: this is just to have an example in that repo

**Review by JalexChen — APPROVED** [review]
LGTM

**Pull request merged**